modules/admin_manual/pages/release_notes.adoc

```diff
@@ -18,7 +19,15 @@
 * xref:changes-in-8-0[Changes in 8.0]
 * xref:changes-in-7-0[Changes in 7.0]
 
-== Changes in 10.1
+== Changes in 10.1.1
+
+ownCloud Server 10.1.1 is a hotfix follow-up release that takes care of https://github.com/owncloud/core/issues/34851[an issue with loading updated apps]. Instead of updating the app versions to their new values in the database, the old version value is written causing the process to repeat with every request.
+
+This issue can cause high load on the database, especially in large installations. If you have already upgraded to 10.1.0, please consider upgrading to 10.1.1. You can expect minimal downtime as it's not necessary to run `occ upgrade` after updating the code to the new version.
+
+Apart from this fix, please consider the ownCloud Server 10.1.0 release notes.
```

Comment:
should we point to https://github.com/owncloud/docs/pull/829 after it's merged ?
at least to point at the documentation update in regards to enterprise bundle updates and preservation of already updated apps